### DISCUSSION TITLE
Truenas Scale deploy app_api docker daemon?

### DISCUSSION THREAD

**Discussion by jc83419** [nextcloud/app_api#283] [Q&A]
HI there,
I m using Truenas Scale deployed Nextcloud. Is there a way I can deploy AppAPI daemon on it?  

Thank you.

**Comment by andrey18106** [comment]
@jc83419 Hi! Yes, the deploy daemon (docker daemon) can be a remote machine or local, you can setup a docker-socket-proxy (https://github.com/cloud-py-api/docker-socket-proxy) for that where you want ExApps to be deployed, and depending on your networking configuration, in AppAPI you just need to setup a connection to it (register Deploy daemon).
